src/Components/loginformvalidationrules.js

```diff
@@ -0,0 +1,21 @@
+/* Form Validation */
+
+// Validates username
+export function validateUsername(values) {
```

Comment:
Do you have validation for password?  Like they must have a password or that the password has to be a certain length etc?